app/src/main/java/com/keylesspalace/tusky/db/ConversationsDao.kt

```diff
@@ -18,17 +18,18 @@ package com.keylesspalace.tusky.db
 import androidx.paging.DataSource
 import androidx.room.*
 import com.keylesspalace.tusky.components.conversation.ConversationEntity
+import io.reactivex.Completable
 
 @Dao
 interface ConversationsDao {
     @Insert(onConflict = OnConflictStrategy.REPLACE)
     fun insert(conversations: List<ConversationEntity>)
 
     @Insert(onConflict = OnConflictStrategy.REPLACE)
-    fun insert(conversation: ConversationEntity)
+    fun insert(conversation: ConversationEntity): Completable
```

Comment:
If we don't want to cancel them (I think we don't) we could as well leave them as blocking ones.